.npmignore

```diff
@@ -1,5 +1,7 @@
-examples/
-src/
+examples/*
+!examples/js/
+src/*
+!src/extras/
```

Comment:
You are absolutely right. Don't know how I missed it so far. will update the PR to include only the examples/js and then it also matches the Bower package exactly. Thanks for pointing out.